src/api/like/content-types/like/schema.json

```diff
@@ -17,15 +17,15 @@
       "relation": "oneToOne",
       "target": "plugin::users-permissions.user"
     },
-    "objectId": {
-      "type": "uid"
```

Comment:
Oh we didn't catch this during code review.
@nikita-pardeshi-github let's also test the like collection.